Description: Fixed package dependencies
 .
 openscribe (1.3-1~bionic1) bionic; urgency=medium
 .
   * Fixed package dependencies to also accept libsox3 instead of just libsox2
Author: Matt Pharoah <mtpharoah@gmail.com>
Last-Update: 2018-05-25

--- openscribe-1.3.orig/changelog.hpp
+++ openscribe-1.3/changelog.hpp
@@ -75,14 +75,22 @@ class ChangelogWindow : public Gtk::Wind
 
 		// put other versions here, with newer versions at the top
 
-		/* Version 1.2.0 */
-		if (since < Version(1,2,1)) {
-			addChangelogSection("1.2.1");
+		/* Version 1.3.0 */
+		if (since < Version(1,3,1)) {
+			addChangelogSection("1.3.1");
+
+			addChangelogEntry("Fixed package dependencies to also accept libsox3 instead of just libsox2");
+		}
+		
+		/* Version 1.3.0 */
+		if (since < Version(1,3,0)) {
+			addChangelogSection("1.3.0");
 
 			addChangelogEntry("Released for Ubuntu 18.04");
 			addChangelogEntry("Switched from gksu to pkexec");
 		}
 		
+		/* Version 1.2.0 */
 		if (since < Version(1,2,0)) {
 			addChangelogSection("1.2.0");
 
--- openscribe-1.3.orig/version.hpp
+++ openscribe-1.3/version.hpp
@@ -47,7 +47,7 @@ struct Version {
 	}
 };
 
-static const Version CURRENT_VERSION = Version{1,2,1};
+static const Version CURRENT_VERSION = Version{1,3,1};
 
 std::ostream &operator<<(std::ostream &out, const Version &ver);
 std::istream &operator>>(std::istream &in, Version &ver);
